collections-csharp/IterateThroughDictionary/IterateThroughDictionary/PerformAnalysisTest.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace IterateThroughDictionary
+{
+    public class PerformAnalysisTest
+    {
+        private Dictionary<int, string> testValues = new Dictionary<int, string>();

```

Comment:
Private fields should have an underscore _. So this private field should be named _testValues. All the other private fields in other classes as well. I can't invest too much time teaching you naming conventions for variables. Please do the research and learn how to name the variables properly.

Reply:
Done, will keep note for next time